packages/msk-insight-commons/rollup.config.ts

```diff
@@ -0,0 +1,56 @@
+import typescript from 'rollup-plugin-typescript2';
```

Comment:
config mostly copied from `react-mutation-mapper`. We don't need many of this dependencies considering we have very few components in `msk-insight-commons` for now